aws_topology/tests/json/apigateway/get_stages_074f0b3.json

```diff
@@ -12,12 +12,9 @@
             },
             "createdDate": 1483996738,
             "lastUpdatedDate": 1502091405,
-            "tags": [
-                {
-                    "key": "StageTagKey1",
-                    "value": "StageTagValue1"
-                }
-            ]
+            "tags": {
+                "StageTagKey1": "StageTagValue1"
+            }
```

Comment:
This needs integration testing - API response is different to what is in boto3 docs

Reply:
API response was wrong, leaving it like this